src/main/java/qna/domain/Answer.java

```diff
@@ -82,47 +95,54 @@ public Question question() {
 
 	public void question(Question question) {
 		this.question = question;
-	}
-
-	public String contents() {
-		return contents;
+		updatedAtNow();
 	}
 
 	public boolean isOwner(User writer) {
 		return this.writer.equals(writer);
 	}
 
+	public void validateIsOwner(User loginUser) throws CannotDeleteException {
```

Comment:
```suggestion
	public void validateIsOwner(User loginUser) {
```

CannotDeleteException을 unchecked exception으로 변경하는게 좋겠네요

Reply:
의아한 부분 중 하나였습니다. 🤔 
Unchecked Exception으로 처리하면 좋겠다는 생각을 갖고 있었습니다... 💭 
다만 기존의 소스 코드를 위배해선 안된다는 고정관념이 뇌리에 박혀있었던 것을 자각했습니다! 💡 
주제가 리팩터링 🏭 인데, 이 부분이 옳지 않았다면 고치는 것 🔧 이 옳았습니다!
감사합니다!!! 🙇